src/quo2/foundations/colors.cljs

```diff
@@ -121,6 +121,11 @@
 (def white-opa-90 (alpha white 0.9))
 (def white-opa-95 (alpha white 0.95))
 
+(def magenta "#EC266C")
```

Comment:
you can remove these colors, we already have them defined. They are also a more dynamic set of colors based on the users customization-color (set in the onboarding phase)

instead when you go to use this `customization-color` you should pass as 
```
(:require [src/quo2/foundations/colors.cljs :as colors])
...
(let [{:keys [customization-color]} (rf/sub [:profile/multiaccount])]
   [quo/button {
    :customization-color (colors/custom-color customization-color 50 40)
   } 
```
that's what it will be for the use case you are using it for anyway where 40 is 40% opacity. 
50 there is a suffix that is used in the mapping, you can see it if you trace through the custom-color function etc

Reply:
if you search in the codebase you'll see many other examples of `customization-color`

Reply:
Thanks @J-Son89 it's changed.